tests/test_util/aiohttp.py

```diff
@@ -249,7 +249,7 @@ async def text(self, encoding="utf-8"):
         """Return mock response as a string."""
         return self.response.decode(encoding)
 
-    async def json(self, encoding="utf-8"):
+    async def json(self, encoding="utf-8", content_type=None):
```

Comment:
is this used ?

Reply:
The library sets it. w/o that there it blows up.